src/experiment/experiment.js

```diff
@@ -119,348 +111,346 @@ export function buildExperiment() {
     ],
```

Comment:
Move out the task Info. Looks like we're not using it so just comment it out for now.